### ISSUE TITLE
Landscape mode 

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-android#798]
A few users have requested landscape mode for the app

**Comment by llelectronics** [comment]
Makes a lot of sense on tablets. I like to use my tablet with a keyboard in landscape mode and its totally counter productive to need to flip it around

**Comment by jparthum** [comment]
Yes, the absence of landscape mode is a deal-breaker for me (and my son).  For now I've uninstalled the app and returned to using a browser.

**Comment by joelmaxuel** [comment]
I would appreciate landscape mode for the app to be used on an unlocked ATV device (Nvidia SHIELD's Darcy).
